ocaml/idl/datamodel.ml

```diff
@@ -84,9 +84,12 @@ let dundee_release_schema_minor_vsn = 93
 let ely_release_schema_major_vsn = 5
 let ely_release_schema_minor_vsn = 100
 
+let falcon_release_schema_major_vsn = 5
+let falcon_release_schema_minor_vsn = 101
```

Comment:
Or should we use a larger number, so that potential increments on Ely during its ten-year life will not reach the Falcon number?